src/ui.tsx

```diff
@@ -423,7 +439,7 @@ function PlanPreview(
           <path
             key={i}
             d={line.reduce((m, {x, y}, j) => m + `${j === 0 ? "M" : "L"}${x} ${y}`, "")}
-            style={i % 2 === 0 ? {stroke: "rgba(0, 0, 0, 0.3)", strokeWidth: 0.5} : {}}
+            style={i % 2 === 0 ? {stroke: "rgba(0, 0, 0, 0.3)", strokeWidth: 0.5} : { stroke: "rgba(0, 0, 0, 0.8)", strokeWidth: state.planOptions.penStrokeWidth * 10 }}
```

Comment:
I'm not sure why this `* 10` is needed... won't this make the width seem much bigger than was requested?

If this is to convert from mm to steps, it should be `* Device.Axidraw.stepsPerMm`

Reply:
To be honest I've just solved this visually. ;) I didn't fully understand how units are converted in saxi. 10 just seemed to worked out. But I guess using `Device.Axidraw.stepsPerMm` makes sense then.  

I've tested with this file setting width and height to 100 and margin to 0.
[test-stroke-width.svg.zip](https://github.com/nornagon/saxi/files/6103361/test-stroke-width.svg.zip)

Then with stroke `width = 1` you get this - strokes of lines closer together (1mm distance) are just meeting:
<img width="889" alt="grafik" src="https://user-images.githubusercontent.com/546852/110364622-4a714200-8044-11eb-8157-92035540e197.png">

with stroke `width = 10` you get this  - strokes of lines further apart (10mm distance) are just meeting:
<img width="889" alt="grafik" src="https://user-images.githubusercontent.com/546852/110364650-53621380-8044-11eb-9129-b74180c09a29.png">